src/runtime/internal/dom.ts

```diff
@@ -307,6 +330,16 @@ export class HtmlTag {
 		this.n = Array.from(this.e.childNodes);
 	}
 
+	l(nodes: any) {
+		this.n = this.n.map(n => {
+			if (n.nodeType === 3) {
+				return claim_text(nodes, (n as Text).data);
+			 } else {
```

Comment:
it looks like this line and the closing curly brace two lines later both have an extra space